api/Document.json

```diff
@@ -4659,7 +4659,9 @@
               "prefix": "webkit"
             },
             "safari_ios": {
-              "version_added": false
+              "version_added": "12",
+              "prefix": "webkit",
+              "notes": "Only available on iPad, not on iPhone."
```

Comment:
For other similar notes about platform support we've added `partial_implementation` false, can you do that here too?